examples/accessibility.html

```diff
@@ -24,23 +24,22 @@
   <link type="text/css" href="styles/examples.css" rel="stylesheet" />
 
   <!-- The following libraries and polyfills are recommended to maximize browser support -->
-  <!-- Include prismatic.js for Magic Leap support: -->
-  <!--<script src="../node_modules/@magicleap/prismatic/prismatic.min.js"></script>-->
-
-  <!-- Include the Custom Elements ES5 adapter shim if you are loading an IE11-compatible bundle -->
-  <!--<script src="../node_modules/@webcomponents/webcomponentsjs/custom-elements-es5-adapter.js"></script>-->
 
-  <!-- Web Components polyfill is required to support Edge and Firefox < 63: -->
+  <!-- 🚨 REQUIRED: Web Components polyfill to support Edge and Firefox < 63 -->
```

Comment:
Firefox 63 should be out of our support range -- also I thought Firefox (at least 64+) fully supports CE/Shadow DOM? (Matrix in README says otherwise)

Reply:
Technically it is, but also technically we still test it. If we are happy to remove it from our test matrix, I'm happy to rescind claims of support. But, there doesn't seem to be an urgency to do so at this moment.

Reply:
Firefox 63 appears to support both [custom elements v1](https://caniuse.com/#feat=custom-elementsv1) and [Shadow DOM v1](https://caniuse.com/#feat=shadowdomv1) according to caniuse

Reply:
Yeah, sorry, this comment says `Firefox < 63`